src/status_im/subs/wallet/activities.cljs

```diff
@@ -11,7 +13,11 @@
  :<- [:wallet/all-activities]
  :<- [:wallet/current-viewing-account-address]
  (fn [[activities current-viewing-account-address]]
-   (filter (fn [{:keys [sender recipient]}]
-             (or (= sender current-viewing-account-address)
-                 (= recipient current-viewing-account-address)))
-           activities)))
+   (let [account-activities (filter (fn [{:keys [sender recipient]}]
```

Comment:
Thanks for your review @ilmotta, been a long time :)

> The code is a bit anticlimactic for Clojure because it's storing individual steps as separate bindings

You are right, thanks for the suggestion, I will update it 👍 

> ;; In src/user.cljs, the inspect function is defined:

I am not sure where this function is? I don't see this function in the codebase. Or do you mean I need to add it?

Reply:
Understood @ilmotta! Thanks for explaining, and I see you made a PR for it already https://github.com/status-im/status-mobile/pull/19927#pullrequestreview-2044717957 🙏

Reply:
Fixed!